freetype/src/harfbuzz/blob.zig

```diff
@@ -13,13 +13,13 @@ pub const Blob = struct {
 
     pub fn init(data: []u8, mode: MemoryMode) ?Blob {
         return Blob{
-            .handle = c.hb_blob_create_or_fail(data.ptr, @intCast(c_uint, data.len), @enumToInt(mode), null, null) orelse return null,
+            .handle = c.hb_blob_create_or_fail(&data[0], @intCast(c_uint, data.len), @enumToInt(mode), null, null) orelse return null,
```

Comment:
FYI this means an empty `data` slice cannot be passed in (panic on accessing index 0). I assume that's what you want / this is intentional? Otherwise `.ptr` would do what you want when `data.len == 0`.

Reply:
yes i just though it's better to panic there